package.json

```diff
@@ -18,11 +21,14 @@
   "dependencies": {
     "@octokit/rest": "^15.9.5",
     "chalk": "^2.4.1",
-    "eslint": "^5.3.0",
     "minimist": "^1.2.0",
     "moment": "^2.22.2",
     "request": "^2.87.0",
     "request-promise-native": "^1.0.5",
     "uuid-validate": "0.0.2"
+  },
+  "devDependencies": {
+    "eslint": "^5.3.0",
+    "semantic-release": "^15.9.8"
```

Comment:
I'd remove this - we're using `npx` above anyway

Reply:
How will the module be available if it isn't installed? Or maybe install it as part of the `travis.yml`?

Reply:
> How will the module be available if it isn't installed?

That's sorta the point of `npx` - running binaries from modules that are not necessarily installed into the project. https://www.npmjs.com/package/npx#description :)